bin/dockerstart

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Check that we're running within docker-compose.
+if [ -z $TOOLFORGE_DOCKER_PORT ]; then
+    echo "This script is only for use via Docker."
+    exit 1
+fi
+
+# Install dependencies.
+composer install -d /var/www || exit 1;
+
+# Start the Lighttpd web server. 
+echo SVG Translate is now at http://localhost:$TOOLFORGE_DOCKER_PORT
+echo Press Ctrl-C to quit.
```

Comment:
hm, if you do `docker-compose up -d` it doesn't stay in the container, which means that to shut down you do `docker-compose down`. That's a fairly common usage, so I don't know if "Press Ctrl-C to quit" would be suitable for that... then again, I imagine people who use that will ... know?

Reply:
That's a good point, I'll remove that bit. The URL is okay though?